### ISSUE TITLE
help - i need more instruction

### ISSUE THREAD

**Issue by andrewarrow** [andrewarrow/go365#1]
If you are feeling stuck on 001 or 002 and can't in good faith move on to the next day because you don't `really` understand everything yet remember to:

1. Be curious - test your hypothesis
2. Add code in your text editor to main.go 
3. Run `go build` and then ./001 or ./002 and run the program
4. Repeat

This loop of coming up with a hypothesis to help clear up why you are confused, writing a new `fmt.Println(thing)` to output the result that will give you a yes/no to your hypothesis, compiling the program (that's the go build step) and then running the program and doing this again and again is your learning process.

And so when you are just starting it might be difficult to get this loop going. Think of each `main.go` file as `your file` and don't worry you can't mess it up. With git there is:

`git checkout main.go`

which will reset it back to it's original version.

And messing it up and exactly what you want to do. Go ahead and try stuff with "reckless abandonment" because when you run `go build` the compiler will tell you exactly what's wrong with the file.

And learning to read the errors from the compiler is very much part of this process.

```
go365/001 $ go build
# go365/001
./main.go:13:5: undefined: b
```

When I see something like this I look at line 13 and read in my head "undefined: b", "undefined: b", OH and then I'll see why the compiler is saying this.

```
go365/001 $ go build
# go365/001
./main.go:13:2: non-bool b1 (type byte) used as if condition
```

and there are a bunch of these different type of compile error messages you'll get to know over time.

But getting these errors, reading them, understanding them, this is part of your hypothesis loop. 

TLDR: type stuff into main.go and try stuff. Over and over.
